opam-ci-check/bin/main.ml

```diff
@@ -186,6 +187,13 @@ let no_transitive_revdeps =
   in
   Arg.value (Arg.flag info)
 
+let use_default_root =
+  let info =
+    Arg.info [ "use-default-root" ]
+      ~doc:"Use the default opam root in ~/.opam instead of creating a new one."
+  in
+  Arg.value (Arg.flag info)
+
```

Comment:
Would it be possible to do this in a separate, preparatory PR?

This is not a huge changeset, but it will impact the whole pipeline, so if we can isolate the pat that will impact the CI as much as possible, that would probably be prudent.